web/client/epics/__tests__/timeline-test.js

```diff
@@ -198,18 +198,6 @@ describe('timeline Epics', () => {
                 flat: [{...STATE_TIMELINE.layers.flat[0], visibility: true}]
             }});
         });
-        it('syncTimelineGuideLayer on changeLayerProperties', done => {
```

Comment:
So these tests were checking for the updated visibility of layers from the state. However, setting the visibility to false was preventing the data from being added to the timeline when a layer is hidden by default on page load; I am now hiding the inactive layers by using the CSS class `'hidden'` instead of using state management; the implementation has been changed making these tests redundant.

Reply:
On a second thought, I could rework the implementation by adding/removing the CSS class that hides the timeline layers by tracking the changing state.